### PR TITLE
bsunanda:Run2-hcx71 Try to sort out some geometry issue of HGCal

### PR DESCRIPTION
**Pull request by bsunanda**
This is to address wrong indexing for finer wafer

### CODE REVIEW

**Comment by bsunanda** [comment]
@cmsbuild Please test

File: Geometry/HGCalCommonData/src/HGCalGeomParameters.cc [hunk 1]
```diff
@@ -420,6 +423,10 @@ void HGCalGeomParameters::loadGeometryHexagon(const DDFilteredView& _fv,
       << cellsf.size() << ":" << cellsc.size() << " wafers " << wafers.size()
       << " layers " << layers.size();
   }
+  if (ntf > 0) { xmf /= ntf; ymf /= ntf;}
+  if (ntc > 0) { xmc /= ntc; ymc /= ntc;}
+  std::cout << "Fine   " << ntf << " x " << xmf << " y " << ymf << std::endl;
+  std::cout << "Coarse " << ntc << " x " << xmc << " y " << ymc << std::endl;
```
Comment (lgray) [inline Geometry/HGCalCommonData/src/HGCalGeomParameters.cc]: Kill the couts.

**Comment by bsunanda** [comment]
@cmsbuild Please test
@lgray the cout statements are removed

**Comment by lgray** [comment]
@bsunanda Thanks!

**Comment by bsunanda** [comment]
@cmsbuild Please test
It failed in DAS query

**Comment by ianna** [comment]
+1

**Comment by davidlange6** [comment]
+1

**Pull request merged**